src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -12,21 +15,63 @@ public UserDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public User create(User entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't save user to db " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public User get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(User.class, id);
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get user by id " + id, e);
+        }
     }
 
     @Override
     public List<User> getAll() {
-        return null;
+        try (Session session = factory.openSession()) {
+            Query<User> getAllCommentQuery = session.createQuery("from User", User.class);
+            return getAllCommentQuery.getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get all users", e);
+        }
     }
 
     @Override
     public void remove(User entity) {
```

Comment:
```suggestion
    public void remove(User user) {
```
same with other naming

Reply:
done